widgets/SpeechToText-transform.json

```diff
@@ -62,6 +69,23 @@
               "en-US"
             ],
             "default" : "en-US"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Transcript Parts Field",
+          "name": "transcriptionPartsField",
+          "widget-attributes" : {
+            "placeholder" : "Field name to use to store all the different chunks of transcription",
```

Comment:
there is no need for a placeholder if there is a default. That being said, I think it makes more sense to have a default for the transcription text field and leave this one empty.